docs/features/models/ollama.md

```diff
@@ -70,6 +70,35 @@ for chunk in model.stream("Write a short story about a cat"):
     print(chunk) # 'In...'
 ```
 
+Additionally, you can use `Ollama` with the `Vision` input if you're running a vision model such as qwen2.5vl. For instance:
+
+```python
+import io
+import requests
+import PIL
+import ollama
+import outlines
+from outlines.templates import Vision
```

Comment:
Note for later: we'll probably need to add this to a different module (like `prompt`)